.gitlab-ci.yml

```diff
@@ -54,237 +63,170 @@
     variables:
       - $SIMP_FULL_MATRIX
 
-stages:
-  - validation
-  - unit
-  - acceptance
-  - deploy
-
-# Puppet 4.7 for PE 2016.4 LTS Support (EOL: 2018-10-21)
-# See: https://puppet.com/misc/puppet-enterprise-lifecycle
+# Puppet + testing environments
 # --------------------------------------
-pup4_7-validation:
-  stage: validation
-  tags:
-    - docker
-  image: ruby:2.1
+.pup_prev: &pup_prev
```

Comment:
Trying to convert these YAML anchors and job names to something generic and versionless like `_current`/`_prev` (instead of the Puppet versions they actually set) is not a good idea for lots of reasons—see discussion in [pupmod-simp-ima#4](https://github.com/simp/pupmod-simp-ima/pull/4#pullrequestreview-170929349)